docs/contributing/testing/Writing-a-golden-file-test-for-package-flutter.md

```diff
@@ -50,6 +51,88 @@ Notice, Gold may wrongly attribute blame for image changes on the dashboard. Pos
 
 ## Creating a New Golden File Test
 
+<!--
+  Note: These instructions are also referred to in the flutter/engine repository
+  ensure that when updating them, the workflow is either consistent or the
+  differences are clearly called out.
+-->
+
+Writing a _new_ golden file test is an involved process, particularly if the
+_test suite_ (sometimes called "task" or "shard") is not already configured for
```

Comment:
> involved process

Seems kind of strange to phrase this this way, especially since it isn;t if the test is already in a covered shard.